doc/shell_guide.md

```diff
@@ -0,0 +1,853 @@
+delete_testcase_after_each_execution_yn=false
+
+# Check disk space before test.
+# If there is not enough disk space, the test tool will send an email to the owner.
+# The owner should clean the disk when he receives this email.
+# The test will continue untill the free disk space reaches the expected amount.
+enable_check_disk_space_yn=true
+owner_email=cui.man@navercorp.com
+
+# set test result feed back type: file or database
+feedback_type=file
+feedback_notice_qahome_url=http://192.168.1.86:8080/qaresult/shellImportAction.nhn?main_id=<MAINID>
+
+git_user=cubridqa
+git_email=dl_cubridqa_bj_internal@navercorp.com
+git_pwd=GITPASSWORD
+
+#these parameters are used when feedback_type=database
+feedback_db_host=192.168.1.86
+feedback_db_port=33080
+feedback_db_name=qaresu
+feedback_db_user=dba
+feedback_db_pwd=
+```
+### Excluded List  
+The cases in the excluded list will not be run in the test.  
+If the case will block the test (eg. it hangs in regression test and the issue will not be fixed recently), we should add the case to the excluded list. For shell test, we have two excluded list files:
+```
+shell/config/daily_regression_test_excluded_list_linux.conf
+shell/config/daily_regression_test_excluded_list_windows.conf
+```
+For example, if we need add case `'shell/_06_issues/_18_1h/bug_bts_12583'` in linux excluded list, we should add these lines in file `'shell/config/daily_regression_test_excluded_list_linux.conf'`.  
+```
+#CBRD-21358 (add comment in this line to record the reason of adding this case in the exclude list)
+shell/_06_issues/_18_1h/bug_bts_12583
+```
+
+### Execute Shell Test
+Login controller node and run CTP:
+```
+cd ~/CTP/bin
+nohup ./ctp.sh shell -c ~/CTP/conf/shell.conf &
+```
+
+### Check Test Results
+Test results can be found in these ways:
+1. the screen output (or file nohup.out)
+2. feedback.log
+3. qa homepage
+
+#### feedback_type=file
+If we set `feedback_type=file`, we can use file `'feedback.log'` to check the test results.  
+
+#### feedback_type=database 
+If we set `feedback_type=database`, we need check the test results on qahome page.  
+For details, please refer to ['4.1 Verify regression test results'](#4.1-Verify-regression-test-results).  
+This is the way we used in regression test.
+
+#### Log File Introduction
+|feedback_type=file|feedback_type=database|description|
+|---|---|---|
+|feedback.log|NA|Records the test result for each case, and the result summary. This file dose not exist if feedback_type=database.|
+|test_status.data|NA|Records the current test status, such as total_case_count, total_fail_case_count. This file dose not exist if feedback_type=database.|
+|main_snapshot.properties|main_snapshot.properties|Records the snapshot of all the parameters used in this test.|
+|check_local.log/check_106.log|check_local.log/check_106.log|Records the log of checking the environment.|

```

Comment:
Change format as below in order fit table 

As is:
`check_106.log|check_local.log`

To be:
```
check_106.log  
check_local.log
```